gmojiAndroid/srcJava/cc/openframeworks/gmoji/GmojiActivity.java

```diff
@@ -101,7 +101,7 @@ protected void onResume() {
 
 		// Make sure device doesnt sleep
 		PowerManager pm = (PowerManager) getSystemService(Context.POWER_SERVICE);
-		wl = pm.newWakeLock(PowerManager.FULL_WAKE_LOCK, "GMOJI");
+		wl = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "GMOJI");

```

Comment:
Yeah should probably get away from wake lock.  Probably make it like FULLSCREEN but without all the kiosk/lockdown features  https://github.com/danzeeeman/FULLSCREEN/blob/FULLSCREEN2.0/app/src/main/java/com/xpo/fullscreen/BASE.java#L31